packages/extra/monitoring/templates/grafana/grafana.yaml

```diff
@@ -68,6 +68,7 @@ spec:
     metadata:
       annotations:
         kubernetes.io/ingress.class: "{{ $ingress }}"
+        acme.cert-manager.io/http01-ingress-class: "{{ $ingress }}"
```

Comment:
I was sure that the ingress class would be taken from `ingressClassName` field. Are you sure that it is necessary?

Reply:
Yeah, I also tried to omit ingressClass from cluster issuer, unfortunately it didn't work.

Reply:
> ingressClass from cluster issuer, unfortunately it didn't work.

it should be only in Cluster Issuer, right?

Reply:
Dunno, when cluster issure missing the section

```yaml
    solvers:
    - http01:
        ingress:
          class: nginx
```

cert-manager reports an error:
```
Failed to determine a valid solver configuration for the set of domains on the Order: no configured challenge solvers can be used for this challenge
```